### PR TITLE
Add some options to coverage tool

### PR DESCRIPTION
**Pull request by dougfelt**
Modified the coverage tool to provide the options of dumping the charmap as a range list, and as a text file, to make it easier to generate some sample text to view.

### CODE REVIEW

File: nototools/coverage.py [hunk 1]
```diff
@@ -18,62 +18,117 @@
...
+  for char in chars:
+    try:
+      name = unicode_data.name(char)
+    except ValueError:
+      name = "<Unassigned>"
+    print "U+%04X %s" % (char, name)
+
+
+def _write_char_text(chars, filepath, chars_per_line):
+  text = [unichr(cp) for cp in chars if not unicode_data.category(cp)[0] in ['M','C','Z']]
```
Comment (jamesgk) [inline nototools/coverage.py]: This and a few other lines are >80 characters.

File: nototools/coverage.py [hunk 2]
```diff
@@ -18,62 +18,117 @@
...
+  filename, _ = path.splitext(path.basename(filepath))
+  m = re.match(r'(.*)-Regular|Bold|Italic|BoldItalic', filename)
```
Comment (jamesgk) [inline nototools/coverage.py]: Should this be `r'(.*)-(?:Regular|Bold|Italic|BoldItalic)'`?

File: nototools/coverage.py [hunk 3]
```diff
@@ -18,62 +18,117 @@
...
+  if m:
+    filename = m.group(1)
+  filename += '_chars.txt'
+  print 'writing file: %s' % filename
+  print '%d characters (of %d)' % (len(text), len(chars))
+  if chars_per_line > 0:
+    lines = []
+    for n in range(0, len(text), chars_per_line):
+      substr = text[n:n+chars_per_line]
+      lines.append(' '.join([cp for cp in substr]))
```
Comment (jamesgk) [inline nototools/coverage.py]: Also don't need to actually generate a list here. Might be more efficient in general to just pass in the generator? Not that it really matters anyhow.

File: nototools/coverage.py [hunk 4]
```diff
@@ -18,62 +18,117 @@
...
+                      action='store_true')
+  parser.add_argument('--chars_per_line', help='Format text in lines of at most this '
+                      'many codepoints,  0 to format as a single line',
+                      type=int, metavar='N', default=32)
+  args = parser.parse_args()
+
+  if not (args.ranges or args.text or args.info):
+    args.info = True
+
+  """Outputs the character coverage of all fonts given on the command line."""
```
Comment (jamesgk) [inline nototools/coverage.py]: Seemed kind of odd to have a docstring down here.

File: nototools/coverage.py (hunk 1 above)
Comment (dougfelt) [inline nototools/coverage.py]: fixed

File: nototools/coverage.py (hunk 2 above)
Comment (dougfelt) [inline nototools/coverage.py]: Yes, of course.  Thanks.

File: nototools/coverage.py (hunk 3 above)
Comment (dougfelt) [inline nototools/coverage.py]: Thanks, I just need to learn to do this.

File: nototools/coverage.py (hunk 4 above)
Comment (dougfelt) [inline nototools/coverage.py]: Some kind of editing error, not sure how that happened.

**Comment by jamesgk** [comment]
LGTM

**Pull request merged**